homeworks/modosandras/homework-02/fictive_character.py

```diff
@@ -0,0 +1,11 @@
+name = input("Név: ")
+age = input("Életkor: ")
```

Comment:
nem hiba, de az int() típuskonverziót akár itt is meghívhatod, és akkor nem kell a 7.soron kizárólag ezt csinálni.